frontend/lib/i18n/locales/nl.ts

```diff
@@ -28,7 +28,7 @@ export const nl = {
   loading: "Laden...",
 
   "candidates_votes.check_totals":
-    "Controleer het totaal van deze lijst. Overleg met coördinator als het papier niet is ingevuld.",
+    "Controleer het totaal van de lijst. Overleg met de coördinator als op het papier niets is ingevuld",
```

Comment:
Ik denk dat de punt na de tweede zin toch moet blijven, maar ik durf dat niet met zekerheid te zeggen.

Reply:
Hier heeft @jorisleker een mening over ;)

Reply:
Zoals ik het heb geleerd kan de punt wegblijven als het (een bulletpoint met) 1 zin is, maar zodra er meer zinnen komen is het raar om de punt inene weg te laten. Maar ik hoor graag de mening van @jorisleker :)

Reply:
Wat @marjoleintamis zegt! Voor de punten en komma's volg ik graag de taalexpert :-)

Reply:
Fixed in faeb61669024d3ce334f6fc3abccdd4860de9505